### PR TITLE
Print fizz buzz

### PR DESCRIPTION
**Pull request by mfellner**
The program should print fizz, buzz.

### CODE REVIEW

File: hello.py [hunk 1]
```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/env python
 
-print("Hello, World!")
+for i in range(0,10):
+    if (i % 3):
```
Comment (prayerslayer) [inline hello.py:4]: I think you are missing `== 0` here.

Comment (mfellner) [inline hello.py]: Sorry, I don't understand. Could you elaborate?

Comment (prayerslayer) [inline hello.py]: You want to check if `i` is divisible by 3 and this you can do with `i % 3 == 0` (0 is coerced to False). Does this make sense to you?

Comment (mfellner) [inline hello.py]: We created a public channel for you at [gitter.im/mfellner/hello-world-pr6-fizz-buzz](gitter.im/mfellner/hello-world-pr6-fizz-buzz)
Join the chat at https://gitter.im/zalando/zappr-mock

**Comment by mfellner** [comment]
Hello, this is a comment!

**Comment by mfellner** [comment]
Test!

**Comment by mfellner** [comment]
Test!

**Comment by mfellner** [comment]
Yay!

**Comment by mfellner** [comment]
kkkk

**Pull request closed without merging**